constructors.js

```diff
@@ -72,6 +101,14 @@
    * @param  {number} damage  Amount of damage to deal to the spellcaster
    */
 
+  this.inflictDamage = function (damage) {
```

Comment:
This should be a method on the Spellcaster's prototype. Same for the following two methods.